### PR TITLE
Review Trial Project PR

### PR DESCRIPTION
**Pull request by Kossi-stack**
**This project is to review one of my past projects during trials**
It is a tribute page that contains two sections a header, the main page, and  CSS to style it.

- Added index.html file and style.css file,
- Added test.md file,
- and added test.md file to .gitignore file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    <footer>
+        <h3>
+            If you have time, you should read more about this incredible human being on his <a
+                href="https://en.wikipedia.org/wiki/Norman_Borlaug" id="tribute-link" target="_blank"
+                rel="noopener noreferrer">Wikipedia entry.</a>
+        </h3>
+    </footer>
+
+</body>
+
```
Comment (Kossi-stack) [inline index.html:128]: Please do not add blank lines or spaces without a reason. Add blank lines to separate large or logical code blocks. 💪

File: style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  }
+
+  img {
+    max-width: 100%;
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+  }
+}
```
Comment (Kossi-stack) [inline style.css:86]: I think spaces and indentations are being well structured for readability. 💯

**Review by Kossi-stack — COMMENTED** [review]
Congratulations Good job 🎉 

**Required Changes ♻️**

Please check my comments below and make the required changes.

Submit for a re-review when you are done.

Thanks 😁

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @Kossi-stack :tada: :clinking_glasses: 

_You've used the GitHub `Review Changes` and `inline comments` as required by this project. Also, you've clearly stated the reasons for your comments in such a polite manner 🤝_

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

Again, congratulations, you can merge to the default branch.
Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**